adagrams/game.py

```diff
@@ -1,13 +1,128 @@
 from random import randint
 
+LETTER_POOL = {
+    1: 'A', 2: 'A', 3: 'A', 4: 'A', 5: 'A', 6: 'A', 7: 'A', 8: 'A', 9: 'A',
+    10: 'B', 11: 'B',
+    12: 'C', 13: 'C',
+    14: 'D', 15: 'D', 16: 'D', 17: 'D',
+    18: 'E', 19: 'E', 20: 'E', 21: 'E', 22: 'E', 23: 'E', 24: 'E', 25: 'E', 26: 'E', 27: 'E', 28: 'E', 29: 'E',
+    30: 'F', 31: 'F',
+    32: 'G', 33: 'G', 34: 'G',
+    35: 'H', 36: 'H',
+    37: 'I', 38: 'I', 39: 'I', 40: 'I', 41: 'I', 42: 'I', 43: 'I', 44: 'I', 45: 'I',
+    46: 'J',
+    47: 'K',
+    48: 'L', 49: 'L', 50: 'L', 51: 'L',
+    52: 'M', 53: 'M',
+    54: 'N', 55: 'N', 56: 'N', 57: 'N', 58: 'N', 59: 'N',
+    60: 'O', 61: 'O', 62: 'O', 63: 'O', 64: 'O', 65: 'O', 66: 'O', 67: 'O',
+    68: 'P', 69: 'P',
+    70: 'Q',
+    71: 'R', 72: 'R', 73: 'R', 74: 'R', 75: 'R', 76: 'R',
+    77: 'S', 78: 'S', 79: 'S', 80: 'S',
+    81: 'T', 82: 'T', 83: 'T', 84: 'T', 85: 'T', 86: 'T',
+    87: 'U', 88: 'U', 89: 'U', 90: 'U',
+    91: 'V', 92: 'V',
+    93: 'W', 94: 'W',
+    95: 'X',
+    96: 'Y', 97: 'Y',
+    98: 'Z',
+}
+
+MAX_HAND_SIZE = 10
+MIN_LETTER_OCCURANCE = 0
+MAX_LETTER_OCCURANCE = 98
+
+SCORE_CHART = {
+    'A': 1, 'E': 1, 'I': 1, 'O': 1, 'U': 1,
+    'L': 1, 'N': 1, 'R': 1, 'S': 1, 'T': 1,
+    'D': 2, 'G': 2,
+    'B': 3, 'C': 3, 'M': 3, 'P': 3,
+    'F': 4, 'H': 4, 'V': 4, 'W': 4, 'Y': 4,
+    'K': 5,
+    'J': 8, 'X': 8,
+    'Q': 10, 'Z': 10
+}
+
 def draw_letters():
-    pass
+    remaining_letters =[]
+    for i in range(1,99):
+        remaining_letters.append(LETTER_POOL[i])
+    hand = []
+    while len(hand) < MAX_HAND_SIZE:
+        new_letter_value = randint(0, len(remaining_letters) -1)
+        new_letter = remaining_letters[new_letter_value] 
+        hand.append(new_letter)
+        remaining_letters.remove(new_letter)
+    return hand
 
 def uses_available_letters(word, letter_bank):
-    pass
+    word = capitalize_word(word)
+    for letter in word:
+        if letter not in letter_bank:
+            return False
+        else:
+            word_letter_frequency = check_letter_frequency(word)
+            hand_letter_frequency = check_letter_frequency(letter_bank)
+            if word_letter_frequency[letter] > hand_letter_frequency[letter]:
+                return False
+            else:
+                all_letters_are_in_hand = True
+    return all_letters_are_in_hand
+        
+def capitalize_word(word):
+     word = word.upper()
+     return word
+
+def check_letter_frequency(x):
+    letter_count = {}
+    for letter in x:
+        if letter not in letter_count:
+            letter_count[letter] = 1
+        else:
+            letter_count[letter] += 1
+    return letter_count
 
 def score_word(word):
-    pass
+        word = capitalize_word(word)
```

Comment:
If you use `capitalize_word` to chance the value referenced by `word`, I'd prefer the variable name to describe the fact that the string has changed. For example "cool" becomes "COOL". 
```suggestion
        capitalized_word = capitalize_word(word)
```